roles/ceph-common/tasks/check_system_and_vars.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: fail on unsupported system
+  fail: "msg=System not supported {{ ansible_system }}"
+  when: "ansible_system not in ['Linux']"
+
+- name: fail on unsupported architecture
+  fail: "msg=Architecture not supported {{ ansible_architecture }}"
+  when: "ansible_architecture not in ['x86_64']"
+
+- name: fail on unsupported distribution
+  fail: "msg=Distribution not supported {{ ansible_os_family }}"
+  when: "ansible_os_family not in ['Debian']"
+
+- name: make sure an installation source was chosen
+  fail: msg="choose an installation source or read https://github.com/ceph/ceph-ansible/wiki"
```

Comment:
Something in this file specifically? Or make a README? That was on my to-do list.